.github/pull_request_template.md

```diff
@@ -18,8 +18,8 @@
 
 <!-- Camunda maintains 18 months of minor versions. Backporting your change to multiple versions is common. -->
 
-- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).
-- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
+- [ ] My changes are for **an upcoming minor release** and are in the `/docs` directory (version 8.8) and/or the `/versioned_docs/version-8.7/` directory.
```

Comment:
```
- [ ] My changes are for **an upcoming minor release** and:
  - [ ] are in the `/docs` directory (version 8.8).
  - [ ] are in the `/versioned_docs/version-8.7/` directory (version 8.7).
  ```
I could see a version where these are separate being helpful, as it asks for a more deliberate selection, but it might be overkill. Either way, I think this will definitely help improve determining where docs go until the minor!

Reply:
This was actually exactly what I was struggling with earlier. I see the PR template as a two-fold communication mechanism - helping contributors know where to put their changes but also helping us review PRs and confirm the changes are where they should be.

I think we should split them because people are pretty bad about using version tags.

Reply:
I like this better, will update.

Reply:
```suggestion
- [ ] My changes are for **an upcoming minor release** and:
  - [ ] are in the `/docs` directory (version 8.8).
  - [ ] are in the `/versioned_docs/version-8.7/` directory (version 8.7).
```